### PR TITLE
Improve clarity and redundancy  of PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by nuno-vieira**
### 🔗 Issue Link
None

### 🎯 Goal
Improve the PR Template. Some points were not clear and some were a bit redundant (Implementation vs Changes).

### 🛠 Implementation
Update the PULL_REQUEST_TEMPLATE.md file.

### 🎨 Showcase
N/A

### 🧪 Testing
N/A

### ☑️ Contributor Checklist
- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [x] This change follows zero ⚠️ policy (required)
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS)

### 🎁  Meme
image

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
+_Provide all Jira tickets and/or Github issues related to this PR, if applicable._
 
 ### 🎯 Goal
 
 _Describe why we are making this change._
 
 ### 🛠 Implementation
 
-_Provide a description of the implementation._
+_Provide a description of the implementation and list the most impportant changes._
```
Comment (nuno-vieira) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
_Provide a description of the implementation and list the most important changes._
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -34,6 +30,6 @@ _Explain how this change can be tested (or why it can't be tested)_
 - [ ] Comparison screenshots added for visual changes
 - [ ] Affected documentation updated (docusaurus, tutorial, CMS)
 
-### 🎉 GIF
+### 🤡 Memes
```
Comment (nuno-vieira) [inline .github/PULL_REQUEST_TEMPLATE.md:33]:
```suggestion
### 🤡 Meme
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,29 +1,25 @@
...
 ### 🎯 Goal
 
 _Describe why we are making this change._
 
 ### 🛠 Implementation
 
-_Provide a description of the implementation._
+_Provide a description of the implementation and list the most important changes._
 
-### 📝 Changes
```
Comment (bielikb) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: Let's keep this in the template. We're leveraging this section when doing a quick summary about changes in PRs that are not carrying the UI changes in them.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,29 +1,25 @@
-### 🔗 Issue Link
-_Jira or Github issue link, if applicable._
+### 🔗 Issue Links
+_Provide all Jira tickets and/or Github issues related to this PR, if applicable._
```
Comment (bielikb) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: 🙌

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -34,6 +30,6 @@ _Explain how this change can be tested (or why it can't be tested)_
 - [ ] Comparison screenshots added for visual changes
 - [ ] Affected documentation updated (docusaurus, tutorial, CMS)
 
-### 🎉 GIF
+### 🤡 Meme
```
Comment (bielikb) [inline .github/PULL_REQUEST_TEMPLATE.md:33]: Could we replace the 🤡 with 🎁 or 🌠 or similar image emoji?
The 🤡 emoji carries a bad connotation with it - mostly on the social media and our daily communications via chat :/
Thanks.

**Review by bielikb — CHANGES_REQUESTED** [review]
Nice one @nuno-vieira. I left 2 comments. otherwise LGTM!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (nuno-vieira) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: I find this a bit redundant and most people just remove this section. And if your implementation can be listed in bullet points, you can do it in the "Implementation section", I even updated the description: **Provide a description of the implementation and list the most important changes** 

But I'm fine with reverting these changes 👍 Thoughts @GetStream/ios-developers  ?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (nuno-vieira) [reply .github/PULL_REQUEST_TEMPLATE.md:33]: Yes, make sense! Nice spot, will change this!

Comment (nuno-vieira) [inline .github/PULL_REQUEST_TEMPLATE.md:33]:
```suggestion
### 🎁 Meme
```

Comment (nuno-vieira) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done! ✅

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (bielikb) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: just ccing one more time -> @GetStream/ios-developers

Comment (evsaev) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: Yes, there's a lot of intersection between changes & implementation sections. I would either go with just `📝 Changes` (as it works for pure documentation PRs) or have both but make `🛠 Implementation` optional.

**Review by polqf — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (nuno-vieira) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: @evsaev @bielikb Can you please re-review? Thanks!

**Review by evsaev — APPROVED** [review]
🚢

**Review by bielikb — APPROVED** [review]

**Pull request merged**